src/binaryen-c.cpp

```diff
@@ -70,6 +70,11 @@ Literal fromBinaryenLiteral(BinaryenLiteral x) {
 static std::mutex BinaryenFunctionMutex;
 static std::mutex BinaryenFunctionTypeMutex;
 
+// Optimization options (default is -Os with debug info)
+static int optimizeLevel = 2;
```

Comment:
maybe there's a way to reuse src/tools/optimization-options.h which has those constants? or to share code with it.

Reply:
That was my first thought as well but then I discovered that `tools/optimization-options.h` has a dependency on `Options` in `support/command-line.h` which felt wrong to include.

Reply:
True. We should probably do a general refactoring of those defaults, probably into `pass.h`. I can do that unless you want to.

Reply:
Currently the defaults are in `support/defaults.h`, which is then included in `binaryen-c.h` and `tools/optimization-options.h`. I'd probably just move them then, so it might be better when you'll take a look.